src/openforms/prefill/attributes_generator.py

```diff
@@ -0,0 +1,122 @@
+    schema: {{ schema }}
+    command: {{ command }}
+    \"\"\"
+
+    {% for attribute in attributes %}{{ attribute.attribute_variable_name }} = "{{ attribute.attribute_value }}", _("{{ attribute.attribute_label }}")
+    {% endfor %}
+"""
+
+
+@dataclass
+class OpenApi3AttributesGenerator:
+    url: str = ""
+    schema: str = ""
+    command: str = ""
+    # TODO features present in the generate_prefill_from_spec command
+    # - Specify a service and then generate the attributes from the OAS of this service
+    # - Specify a path and then only generate the attributes for this path
+    # - If no path/schema are specified, print all paths and schemas in the OAS
+    # - Option to add '_embedded.' in paths for $refs
+    # Additional features that could be good
+    # - Keep track of the titles in addition to the properties names in the schema
+
+    def get_specifications(self):
+        if self.url:
+            # The content types supported in the openapi3-parser are these:
+            # https://github.com/manchenkoff/openapi3-parser/tree/master/src/openapi_parser/home/silvia/repositories/openapi3-parser/src/openapi_parser/enumeration.py
+            # However, in the haal centraal OAS we find content types such as "application/hal+json". So with
+            # strict_enum=False, the parser just checks that the content type is a string
+            return parse(uri=self.url, strict_enum=False)
+
+        raise AttributeGeneratorException(
+            "No URL to the OAS specifications was provided."
+        )
+
+    def get_attributes(self, property_schema):
+        attributes = []
+        if hasattr(property_schema.schema, "properties"):
+            for schema_property in property_schema.schema.properties:
+                nested_attributes = self.get_attributes(schema_property)
+                attributes += [
+                    [property_schema.name] + nested_attribute
+                    for nested_attribute in nested_attributes
+                ]
+        else:
+            attributes.append([property_schema.name])
+        return attributes
+
+    def get_schemas_attributes(self, oas_specifications):
+        # In the specifications, the schemas are under the path "components > schemas"
+        parsed_schemas = oas_specifications.schemas
+
+        if self.schema:
+            if self.schema not in parsed_schemas:
+                raise AttributeGeneratorException(
+                    f'The given schema name "{self.schema}" could not be found in the specifications'
+                )
+
+            return [
+                self.get_attributes(schema_property)
+                for schema_property in parsed_schemas[self.schema].properties
+            ]
+
+        raise AttributeGeneratorException("No schema specified")
+
+    def format_attributes(self, attributes):
+        """Turn each attribute in ``{"attribute_variable_name": "bla_bla", "attribute_value": "bla.bla", "attribute_label": "Bla > Bla"}``"""
```

Comment:
Can you pay a bit more attention to the docstring hygiene, following [PEP 257](https://peps.python.org/pep-0257/)

Essentially, the summary should fit one line (so keeping a max length of ~ 80 chars is a good guideline), followed by a more elaborate explanation.

```suggestion
        """
        Prepare attributes for code generation.
        
        Each attribute is prepared by:
        
        * using snake_case to create valid variable names/identifiers
        * creating a string value with periods as path separators
        * capitalize each part for the human readable label
       """
```